ViAn/GUI/analysisslider.cpp

```diff
@@ -38,23 +39,101 @@ void AnalysisSlider::paintEvent(QPaintEvent *ev) {
 }
 
 /**
+ * @brief AnalysisSlider::set_analysis
+ * Adds all POIs to the slider
+ * @param analysis
+ */
+void AnalysisSlider::set_analysis(Analysis* analysis) {
```

Comment:
is this a slot function? if so it might be good to asssert that the analysis pointer actually points to valid data